types/firestore.d.ts

```diff
@@ -1452,6 +1452,12 @@ declare namespace FirebaseFirestore {
    * API and the underlying Firestore v1 RPCs.
    */
   export const v1: {FirestoreClient: any, FirestoreAdminClient: any};
+
+  /**
+   * Status error code enums used by google-gax that correspond to the error
+   * codes returned by Firestore calls.
+   */
+  export const Status: import('../dev/src/types').Status;
```

Comment:
Two questions:
- Do we need to roundtrip through `src/types`?
- Could we export this type under a different name? `Status` is very generic. It would be better if this was tied to gRPC somehow.

Reply:
Removed the roundtrip. I originally thought there was a rule about importing global packages into an export, but that's only inside an import.

Renamed to GrpcStatus.